appveyor.bat

```diff
@@ -99,9 +99,12 @@ call :downloadfile %LUA_URL% downloads\lua.zip
 7z x downloads\lua.zip -o%LUA_DIR% > nul || exit 1
 
 :: Perl
-call :downloadfile %PERL_URL% downloads\perl.zip
-7z x downloads\perl.zip -oC:\ > nul || exit 1
-for /d %%i in (C:\ActivePerl*) do move %%i C:\Perl%PERL_VER%
+call :downloadfile %PERL_URL% downloads\perl.exe
+mkdir c:\ActivePerl
+start /wait downloads\perl.exe /extract:c:\ActivePerl /exenoui /exenoupdates /quiet /norestart
+pushd c:\ActivePerl
+for /d %%i in (*) do for /d %%j in (%%i\*) do move %%j c:\ActivePerl
+popd

```

Comment:
I feel this part a bit tricky, because you use a double for loop.
After executing `downloads\perl.exe`, it is extracted in `C:\<extract dir>\<random number>`.
It can move with a single for loop if a temporary directory is used. How about the following?

```bat
call :downloadfile %PERL_URL% downloads\perl.exe
mkdir c:\ActivePerlTemp
start /wait downloads\perl.exe /extract:c:\ActivePerlTemp /exenoui /exenoupdates /quiet /norestart
for /d %%i in (C:\ActivePerlTemp\*) do move %%i %PERL_DIR%
```